CHANGELOG.md

```diff
@@ -56,6 +56,12 @@ Added the ability to configure the WASM module used for assets canisters through
 
 ### fix: dfx deploy and icx-asset no longer retry on permission failure
 
+### feat: --created-at-time for the ledger functions: transfer, create-canister, and top-up
+
+### fix: ledger transfer duplicate transaction now returns an error.
+
+Before it was possible that a user could send 2 ledger transfers and both would show success but there would have been only 1 ledger transfer. 
```

Comment:
> Before it was possible that a user could send 2 ledger transfers and both would show success but there would have been only 1 ledger transfer.

What's the scenario in which this description would be the case?  Two calls to `dfx ledger transfer` with identical `timestamp_nanos`?

Reply:
> What's the scenario in which this description would be the case? Two calls to dfx ledger transfer with identical timestamp_nanos? 

Yes.

Reply:
Okay.  It's fine to display that the result from the replica was `TxDuplicate`, but even in that case both `dfx ledger transfer` calls should return success, indicating that the transfer completed exactly once.

Note that according to https://internetcomputer.org/docs/current/developer-docs/integrations/icrc-1/#transaction-deduplication-, this would mean that `created_at_time`, `memo`, the transfer argument fields, and the caller all have the same values.

```
If the ledger observed a structurally equal transfer payload (i.e., all the transfer argument fields and the caller have the same values) at transaction with index i, it should return variant { Duplicate = record { duplicate_of = i } }.
```